ebb/ebb-callback.S

```diff
@@ -0,0 +1,613 @@
+/* Event-Based Branch Facility API.  PowerPC32 callback handler.
```

Comment:
It covers all three ABI's now. That comment should be updated. I've tested and verified the generated code on all three.